### PR TITLE
update dev release docs

### PR DESCRIPTION
**Pull request by millzpaugh**
When @mapsam and I were publishing a dev release of vtuqery with mason-js, our process looked slightly different than the docs, so I added an additional option for referencing a dev release. 

cc @GretaCB

### CODE REVIEW

File: docs/publishing-binaries.md [hunk 1]
```diff
@@ -157,10 +157,25 @@ index e00b7b5..22f7cd9 100644
...
+
+**Option B:** Issue a npm dev release after the binary is published. 
+
+Run `npm publish --tag dev` and then require your module in downstream applications like: 
+
+```js
+"your-module": "0.1.0-alpha1",
+```
+
+For npm dev releases, it’s good to use the `--tag <something>` to avoid publishing to the latest tag. If you run `npm publish` then `0.1.0-alpha` is what anyone running `npm install vtquery --save` will receive.
```
Comment (mapsam) [inline docs/publishing-binaries.md]: can we change `vtquery` to `<your module>`?

File: docs/publishing-binaries.md [hunk 2]
```diff
@@ -157,10 +157,25 @@ index e00b7b5..22f7cd9 100644
 
 3. Publishing C++ binaries by pushing a commit with `[publish binary]` per https://github.com/mapbox/node-cpp-skel/blob/master/docs/publishing-binaries.md#7-all-done
 
-4. Require your module in downstream applications like:
+
+4.
```
Comment (mapsam) [inline docs/publishing-binaries.md]: To fix this formatting, I think you can use the "three spaces as a line break" trick:

Example markdown (note the highlighted _three spaces_ at the end of the option A line):

screen shot 2018-04-06 at 10 27 55 am

Example output: 

screen shot 2018-04-06 at 10 28 03 am

File: docs/publishing-binaries.md (hunk 1 above)
Comment (GretaCB) [inline docs/publishing-binaries.md]: I'm struggling to understand the relationship between
```
--tag <something>
```
and the following examples provided:
- `npm publish --tag dev`
-  `0.1.0-alpha`
- `"0.1.0-alpha1",`

File: docs/publishing-binaries.md (hunk 2 above)
Comment (millzpaugh) [reply docs/publishing-binaries.md]: done!

File: docs/publishing-binaries.md (hunk 1 above)
Comment (millzpaugh) [reply docs/publishing-binaries.md]: done!

Comment (millzpaugh) [reply docs/publishing-binaries.md]: @GretaCB that is confusing - the example package name should just be `0.1.0-alpha` - `alpha1` was a typo. I've updated this!

**Comment by springmeyer** [comment]
@millzpaugh thanks for getting this going! Is it ready to merge now that you've addressed PR feedback?

**Pull request merged**